### DISCUSSION TITLE
grafana pgsql catalog

### DISCUSSION THREAD

**Discussion by lujay** [pgsty/pigsty#250] [General]
![image](https://user-images.githubusercontent.com/6263244/225273083-ee168981-958f-4033-91f2-dc864cfc4583.png)
感觉面板太多了，作为使用者，一般都是基层ops ，一下子看不到主要信息，不知道这些监控数据的用处

**Comment by Vonng** [comment]
很多 Dashboard 主要是给老司机问诊看的，一般研发关注 Overview / Cluster 即可，会列出最核心的指标出来。后续会有一些教程介绍使用详情

**Reply by lujay** [reply]
小白用户表示， 都不知道怎么去连接实例，使用教程对于小白 ，不够流畅呢，我觉得

**Reply by Vonng** [reply]
http://pigsty.cc/zh/docs/pgsql/concept/access/ 试试这个？
